ZAPD/ZFile.h

```diff
@@ -34,7 +34,9 @@ class ZFile
 	std::map<offset_t, Declaration*> declarations;
 	std::string defines;
 	std::vector<ZResource*> resources;
-	uint32_t segment;
+
+	// Default to using virtual addresses
```

Comment:
Shouldn't this say `use` instead of `using`?
```suggestion
	// Default to use virtual addresses
```

Reply:
"default to using x" makes sense, you are actively using that default thing